registry/extension/oras/artifactservice.go

```diff
@@ -72,7 +87,16 @@ func (h *referrersHandler) Referrers(ctx context.Context, revision digest.Digest
 		return nil, err
 	}
 
-	return referrers, nil
+	// sort the list of descriptors that contain the created annotation
+	sort.Slice(referrersSorted, func(i, j int) bool {
+		firstElem, _ := time.Parse(time.RFC3339, referrersSorted[i].Annotations[createAnnotationName])
+		secondElem, _ := time.Parse(time.RFC3339, referrersSorted[j].Annotations[createAnnotationName])
```

Comment:
Should be a `const` as it's used in multiple places

Reply:
Sorry a bit confused which part you are referring to. Is it the `time.RFC3339` that should be a `const`?

Reply:
Yes, I meant that we can define one variable that's used everywhere

Reply:
switched to using a constant